zyte_common_items/pages.py

```diff
@@ -92,6 +97,58 @@ async def currencyRaw(self) -> Optional[str]:
         return None
 
 
+class DescriptionMixin(FieldsMixin):
+    """Provides description and descriptionHtml field implementations."""
+
+    _description_html: Union[HtmlElement, str, None] = None
+    _description_str: Optional[str] = None
+
+    _description_default = False
+    _descriptionHtml_default = False
+
+    async def _get_description(self) -> Optional[str]:
+        if self._description_default:
+            return None
+        if self._description_str is None:
```

Comment:
Should we use another sentinel, not None, for the cases self.description returns None after a long computation?